supervisord.conf

```diff
@@ -0,0 +1,7 @@
+[supervisord]
+#This Should be change application owner
+[program:material-api]
+command=/bin/bash -c "node index.js"
```

Comment:
Shall we change this to `npm start` ?

Reply:
this has been removed since I have added pm2.